src/models/db/caseStatus.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const knex = require('../../lib/knex').knex
+
+const TABLE_NAME = 'N_LU_CASE_STATUS'
+
+const CASE_STATUS_CODES = {}
+const CASE_STATUS_DESCRIPTIONS = {}
+
+function readStatusCodes () {
+  return knex(TABLE_NAME)
+    .select()
+    .then((rows) => {
+      rows.forEach((row) => {
+        CASE_STATUS_CODES[row.CASE_STATUS_ID] = row.STATUS_CODE
+        CASE_STATUS_DESCRIPTIONS[row.STATUS_CODE] = row.CASE_STATUS_ID
+      })
+    })
+}
+
+function getStatusDescription (statusCode) {
+  return CASE_STATUS_CODES[statusCode]
```

Comment:
The logic is correct, I think the variable names are what is confusing. Let me think of better ones.